src/components/Skeleton/__snapshots__/Skeleton.test.ts.snap

```diff
@@ -0,0 +1,62 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Skeleton /> Circle story renders snapshot 1`] = `
+<div
+  aria-hidden="true"
```

Comment:
I wonder if we should have some kind of screen ready-only text like the spinner has because skeleton states are like an alternative to spinners.

Reply:
I think for these ones, they would only be used on a page's initial load, and shown for a very brief moment, so any text or assistive aids should be handled by whatever caused the navigation, triggering an update to some component with `aria-live`. the loading indicator is similar, but would appear where some content used to appear, or overlaying the whole body of a component? Hard to say since we don't have reference designs just yet :S

Reply:
Makes sense with `aria-live`. Am wondering how you're going to communicate such requirement?

Reply:
I thought about it and I think the big difference between the skeleton states and a spinner is there's typically one spinner (well I have seen apps where there are a bunch of spinners but that looks pretty weird) whereas there are more likely to be multiple skeletal state shapes, and we wouldn't want each one of them to have a message announcing that something is loading. I think this might be a similar situation to the `Badge` where we're going to have to leave it to the developer to add something themselves, and maybe the best thing for us to do is just add very visible documentation.